plenum/client/client.py

```diff
@@ -182,6 +190,27 @@ def __init__(self,
         tp = loadPlugins(self.basedirpath)
         logger.debug("total plugins loaded in client: {}".format(tp))
 
+        self._multi_sig_verifier = self._create_multi_sig_verifier()
+        self._bls_node_public_keys = None  # node_name -> bls_key
```

Comment:
Please use BlsKeyRegister class instead of this dict.
Probably we need to provide implementation based on keys from Ledger (we will be able to use implemnetation based on State once we support pool state on client)